endpoints/openrtb2/video_auction.go

```diff
@@ -134,6 +134,10 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 		handleError(labels, w, errL, ao)
 		return
 	}
+	if len(podErrors) > 0 {
+		//remove incorrect pods
+		videoBidReq = cleanupVideoBidRequest(videoBidReq, podErrors)
+	}
 
```

Comment:
This if statement is exactly the same as the one in line 155 below and nothing seems to happen with podErrors between here and there. Are the two checks necessary?

Reply:
Good catch. This appeared because of incorrect merge of merge conflicts. Fixed now.